lib/agent0/examples/interactive_hyperdrive_example.py

```diff
@@ -0,0 +1,104 @@
+"""Example script for using interactive hyperdrive."""
+import datetime
+
+from fixedpointmath import FixedPoint
+
+from agent0.hyperdrive.interactive import InteractiveHyperdrive, LocalChain
+
+if __name__ == "__main__":
```

Comment:
I think you should pull this out of a `__name__ == "__main__"` conditional and instead wrap sections in `#%%` comment blocks so that they can be executed as a jupyter notebook in vscode _or_ as a python script.

Reply:
I think the same about the other examples fwiw. I'd like to lean into the vscode environment because that's what everyone at delv uses. We can should discuss an easier solution for sharing this more broadly, but for now I like being able to run the examples via a notebook in vscode or via `python examples/file.py`.